src/Nethermind/Nethermind.Benchmark/Core/Keccak256Benchmarks.cs

```diff
@@ -57,11 +56,10 @@ public Span<byte> ValueKeccak()
             return Nethermind.Core.Crypto.ValueKeccak.Compute(_a).BytesAsSpan;
         }
 
-
-        [Benchmark]
-        public byte[] HashLib()
-        {
-            return _hash.ComputeBytes(_a).GetBytes();
-        }
+        //[Benchmark]
```

Comment:
let it stay vs remove at all

Reply:
We use another Keccak implementation in `Nethermind.Crypto`. This one is not used.